src/Calendar.vue

```diff
@@ -71,6 +71,10 @@
             this.$root.$on(CHANGE_MONTH, function (payload) {
                 me.currentMonth = payload;
             });
+
```

Comment:
the following code is redundant! Please remove 

`this.$root.$on(EVENT_ADDED, function(payload) {
 me.$emit('eventAdded', payload);
 });`